From 4516d7ca4455a620a109bc31a0be90bbb2feac1a Mon Sep 17 00:00:00 2001
From: Yahan Lu <yahan@iscas.ac.cn>
Date: Wed, 17 Aug 2022 07:23:03 +0000
Subject: [PATCH] Revert "[WATCHLISTS] Add riscv watch"

This reverts commit 776b9eb9d9fce486ff3b8e542091649e66a29bc3.

Reason for revert: Wrong email address

Original change's description:
> [WATCHLISTS] Add riscv watch
>
> Change-Id: I6e4dc69d6f22d3108ae74552b72bcafc0be3db64
> Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3793476
> Commit-Queue: Yahan Lu <yahan@iscas.ac.cn>
> Reviewed-by: Victor Gomes <victorgomes@chromium.org>
> Cr-Commit-Position: refs/heads/main@{#82092}

Change-Id: Ie5a9c4ca42e77f1d07fcbe8fbb521e97a0ae7422
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3834579
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: Victor Gomes <victorgomes@chromium.org>
Reviewed-by: Victor Gomes <victorgomes@chromium.org>
Cr-Commit-Position: refs/heads/main@{#82530}
---
 WATCHLISTS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/WATCHLISTS b/WATCHLISTS
index 91155d11945..6d82fdc0cbf 100644
--- a/WATCHLISTS
+++ b/WATCHLISTS
@@ -130,7 +130,6 @@
     'arm': [
       'v8-mips-ports@googlegroups.com',
       'v8-ppc-ports@googlegroups.com',
-      'v8-risc-v-ports@googlegroups.com',
     ],
     'merges': [
       # Only enabled on branches created with tools/release/create_release.py
-- 
2.35.1

